packages/icons/src/aspect-ratio.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { IconProps } from "./types";
+
+export const AspectRatioIcon = React.forwardRef<SVGSVGElement, IconProps>(
```

Comment:
where is it coming from? @taylornowotny said he designed all icons in 16 size and wanted to redo all radix icons because they are 15, so I am wondering why is this one 15.
maybe Taylor just used the radix icon, but in that case you can just import it from the package, why adding it?

Reply:
Correct, all icons are 16px and part of a standardized component set. See the Icons page of the design system.

Reply:
hmm, not sure where @SaraVieira got the 15px icon from then, maybe our figma is inconsistent somewhere

Reply:
I got it from figma, copied from there, it was 15px idk why

Fixed

Reply:
it may be wrong icon there, like the path could be wrong

Reply:
I scaled and copied